packages/atlas/src/providers/joystream/joystream.hooks.ts

```diff
@@ -1,5 +1,8 @@
+import BN from 'bn.js'
```

Comment:
Sometimes we use default import of `bn.js` and sometimes named. Can we standardize it?